src/Bicep.Cli/Commands/RootCommand.cs

```diff
@@ -116,6 +116,27 @@ private void PrintHelp()
       bicep decompile file.json --force
       bicep decompile file.json --outfile file.bicep
 
+  {exeName} decompile-params [options] <file>
+    Attempts to decompile a parameters .json file to .bicepparam.
+
+    Arguments:
+      <file>        The input file
+
+    Options:
+      --outdir <dir>    Saves the output at the specified directory.
+      --outfile <file>  Saves the output as the specified file path.
+      --stdout          Prints the output to stdout.
+      --force           Allows overwriting the output file if it exists (applies only to 'bicep decompile' or 'bicep decompile-params').
+      --bicep-file      Path to the bicep-file that will be referenced in the using declaration
```

Comment:
I think we should call out here the `--bicep-file` value won't be modified and needs to be relative to the output file.

Reply:
When you won't be modified, isn't that the expected behavior? Why do we need to tell the users?